ci/verify-build.sh

```diff
@@ -67,8 +68,31 @@ test_target() {
     # Test again without default features, i.e. without "std"
     $cmd --no-default-features
     $cmd --no-default-features --features extra_traits
+
+    # For freebsd targets, check with the different versions we support if on
+    # nightly or stable
+    case "$rust-$target" in
+        stable-*freebsd*) do_freebsd_checks=1 ;;
+        nightly-*freebsd*) do_freebsd_checks=1 ;;
+    esac
+    
+    if [ "${do_freebsd_checks:-}" = "1" ]; then
+        for version in $freebsd_versions; do
+            export RUST_LIBC_UNSTABLE_FREEBSD_VERSION="$version"
+            $cmd
+            $cmd --no-default-features
+        done
+    fi
 }
 
+freebsd_versions="\
+11 \
+12 \
```

Comment:
This will test 12 twice, once at line 69-70 and once at 82-83.

Reply:
Do you have any idea how best to avoid this? It wouldn't be that bad if there isn't anything straightforward.

Reply:
The default version of "12" is set in build.rs.  So you can simply leave 12 off of the list.  But then it would have to be manually updated in case the default ever changes.  So if object files are retained between runs, then running `cargo check` twice should be fast enough, I think.

Reply:
That's about the same conclusion I came to, it should be fast enough that it's better to not worry about forgetting to update the default.

Thanks for taking a look at this!